wrappers/rust/icicle-core/src/ntt/mod.rs

```diff
@@ -72,6 +72,18 @@ impl<'a, S: FieldImpl> NTTConfig<'a, S> {
             is_async: false,
         }
     }
+    pub fn default_config_for_device(device_id: usize) -> Self {
```

Comment:
Maybe it's better to call `default_config` and replace the ctx? Just to avoid code duplication

Reply:
hmm create pool and stream and then abandon them? the calls are probably inexpensive, though - let's discuss

Reply:
I just updated the design doc - after rethinking a bit - I guess it's a bad idea to call set_device implicitly - even if documented